arkane/statmech.py

```diff
@@ -1135,3 +1136,58 @@ def determine_rotor_symmetry(energies, label, pivots):
         logging.info('Determined a symmetry number of {0} for rotor of species {1} between pivots {2}'
                      ' based on the {3}.'.format(symmetry, label, pivots, reason))
     return symmetry
+
+class HinderedRotor2D(Mode):
```

Comment:
So I invested a lot of time trying to get this to work in torsion.pyx, but there's a circular import that occurs because HinderedRotor2D and HinderedRotorClassicalND unlike the other rotor functions both need quantum file reading related imports so I decided to locate them in statmech.py where this didn't create a circular import.

Reply:
OK, let's try to make this move as we discussed offline